.devops/code-review-pipelines.yml

```diff
@@ -19,6 +15,14 @@ trigger: none
 pool:
   vmImage: 'ubuntu-latest'
 
+resources:
+  repositories:
+    - repository: pagopaCommons
+      type: github
+      name: pagopa/azure-pipeline-templates
+      ref: refs/tags/v14
+      endpoint: 'pagopa'
```

Comment:
```suggestion
      endpoint: 'io-azure-devops-github-ro'
```

Reply:
Done in [68d6ddb](https://github.com/pagopa/io-functions-pushnotifications/pull/11/commits/68d6ddbb9ae3649043341177aef41b1219590305)